py/preprocess_data.py

```diff
@@ -5,12 +5,15 @@
 pollsters = pd.read_csv("data/pollsters.csv")
 favorability = pd.read_csv("data/favorability.csv")
 questions = pd.read_csv("data/questions.csv")
+xtab_order = pd.read_csv("data/xtab_order.csv")
+
+# convert val_order to int
+xtab_order["val_order"] = xtab_order["val_order"].astype(int)
```

Comment:
nit per our [style guide](https://ubicenter.dev/contributors/code/style.html#pandas)
```suggestion
xtab_order["val_order"] = xtab_order.val_order.astype(int)
```